xml/art_admin_slemicro_selinux.xml

```diff
@@ -0,0 +1,344 @@
+   Each access that is not explicitly allowed by the policy is denied.
+  </para>
+  <para>
+   You can switch between the <literal>enforcing</literal> and
+   <literal>permissive</literal> modes by using the
+   <command>setenforce</command> command. Or you can switch between all SELinux
+   modes by editing the <filename>/etc/selinux/config</filename> configuration
+   file. Changes performed by the <command>setenforce</command> command are
+   valid only until the next reboot. For persistent changes of the SELinux
+   mode, edit the <filename>/etc/selinux/config</filename> configuration file.
+  </para>
+  <para>
+   The <command>setenforce</command> command has the following syntax:
+  </para>
+<screen>&prompt.root;setenforce <replaceable>MODE_ID</replaceable></screen>
+  <para>
+   where <replaceable>MODE_ID</replaceable> is <replaceable>0</replaceable> for the
+   <literal>permissive</literal> mode or <replaceable>1</replaceable> for the <literal>enforced</literal>
+   mode.
+  </para>
+  <para>
+   To verify the mode, run the following command.
+  </para>
+<screen>&prompt.root;getenforce</screen>
+  <para>
+   The command should return <literal>permissive</literal> or
+   <literal>enforced</literal>, depending on the provided
+   <replaceable>MODE_ID</replaceable>.
+  </para>
+  <para>
+   To change the SELiux mode permanently, in the file
+   <filename>/etc/selinux/config</filename> change the value of
+   <literal>SELINUX</literal> to <literal>disabled</literal>, or
+   <literal>permissive</literal>, or <literal>enforced</literal> as follows:
+  </para>
+<screen>SELINUX=disabled</screen>
+  <para>
+   The changes in the file are applied after next reboot.
+  </para>
+  <note>
+   <title>Relabelling your system after switching from the <literal>disabled</literal> mode</title>
+   <para>
+    If you disable SELinux on your system and then enable it later, make sure
+    that you relabel your system. When SELinux is disabled and you perform
+    changes to your file system, the changes are not reflected in the context
+    any more (for example, new files do not have any context), therefore you
+    need to relabel your system by using the <command>restorecon</command>
+    command, using the <literal>autorelabel</literal> boot parameter or by
+    creating a file that will trigger relabelling on the next boot. To create
+    the file, run the folowing command:
+   </para>
+<screen>&prompt.root;touch /etc/selinux/.autorelabel</screen>
+   <para>
+    After reboot, the file <filename>/etc/selinux/.autorelabel</filename> is
+    replaced with another flag file:
+    <filename>/etc/selinux/.relabelled</filename> to prevent relabelling on
+    subsequent reboots.
+   </para>
+  </note>
+ </section>
+ <section xml:id="sec-policy-overview">
+  <title>SELinux policy overview</title>
+  <para>
+   The policy is the key component in SELinux. Your SELinux policy defines
+   rules that specify which objects can access which files, directories, ports,
```

Comment:
```suggestion
   rules that specify which objects can access specific files, directories, ports,
```

Reply:
I think that changing wording in this case would change the meaning a little bit. Those files do not necessarily must be specific to the object, they could be assigned to more objects.